templates/common/infra/terraform/core/gateway/apim-api/apim-api.tf

```diff
@@ -0,0 +1,93 @@
+terraform {
+  required_providers {
+    azurerm   = {
+      version           = "~>3.18.0"
+      source            = "hashicorp/azurerm"
+    }
+    azurecaf  = {
+      source            = "aztfmod/azurecaf"
+      version           = "~>1.2.15"
+    }
+  }
+}
+
+data "azurerm_api_management" "myapim"{
```

Comment:
I think we should just call this "apim" instead of "myapim".  Would that work?

Reply:
Works fine, has been modified.